pkg/addon/policyframework/manifests/managedclusterchart/templates/namespace.yaml

```diff
@@ -1,6 +1,6 @@
 # Copyright Contributors to the Open Cluster Management project
 
-{{- if and (not .Values.onMulticlusterHub) (ne .Values.installMode "Hosted") }}
+{{- if or (not .Values.onMulticlusterHub) (eq .Values.installMode "Hosted") (.Values.args.syncPoliciesOnMulticlusterHub) }}
```

Comment:
```
eq .Values.installMode "Hosted"
```
Is this correct? it was `ne`

Reply:
Oh yes, you are right. I'll change that.